sdk/core/azure_core_amqp/src/receiver.rs

```diff
@@ -100,6 +100,13 @@ impl AmqpReceiverApis for AmqpReceiver {
         self.implementation.credit_mode()
     }
 
+    /// Receives a delivery from the AMQP receiver.
+    ///
+    /// This method returns a single `AmqpDelivery` that can be used to receive a message from the AMQP receiver.
+    ///
+    /// If the receiver options are set to `auto_accept`, the delivery will have already been accepted, and no
```

Comment:
For functions, the syntax is qualified (parens optional, but I prefer them so it's obvious it's a fn member):
```suggestion
    /// If the receiver options are set to [`AmqpReceiver::auto_accept()`], the delivery will have already been accepted, and no
```